From 2b97f663f294103e37c25a5b815764b42797dde4 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Mon, 9 Oct 2023 19:06:22 +0200
Subject: [PATCH 0677/1141] arm64: defconfig: activate lptimer as tick
 broadcast

During the low power modes the generic ARM timer is deactivated, so the
the tick broadcast need to be used in low power mode,
base on LPTIMER3 which is clocked by LSE on STMicroelectronics boards.

Change-Id: Id4d683e5603a44903ad94e9fda92dd7895df0636
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/336463
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index e0ab86d31c00..5b8412cec452 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -8,6 +8,7 @@ CONFIG_BT_HS=y
 CONFIG_BT_LE=y
 CONFIG_CAN_M_CAN=m
 CONFIG_CAN_M_CAN_PLATFORM=m
+CONFIG_CLKSRC_STM32_LP=y
 CONFIG_COUNTER=m
 CONFIG_CRYPTO_DEV_STM32_CRC=m
 CONFIG_CRYPTO_DEV_STM32_CRYP=m
-- 
2.39.2

